examples/UsbMidiConverter/UsbMidiConverter.ino

```diff
@@ -0,0 +1,22 @@
+/*
+ * Simple USBMIDI converter v0.1
+ *
+ * Forwards MIDI messages from USB to MIDI and back
+ * If you short TX and RX pins of arduino, it should act as MIDI loopback
+ *
+ * (c) Tomas 'harvie' Mudrunka 2019
+ */
+
+#include <usbmidi.h>
+
+void setup() {
+  Serial1.begin(31250); //MIDI baudrate
+}
+
+void loop() {
+  //Forward MIDI
+  while(USBMIDI.available()) Serial1.write(USBMIDI.read());
+  USBMIDI.flush();
```

Comment:
The flush flushes outgoing data, I think it would make sense to swap the flush calls, first flush Serial1, then USBMIDI, just after writing new data into them.

Reply:
good point!